### PR TITLE
Update HelpScreen styling

### PR DESCRIPTION
**Pull request by lubej**
## Links

https://www.loom.com/share/eb53547e59c24c2bbaae399c88055998

### CODE REVIEW

**Comment by buberdds** [comment]
Does TapIcon have a correct color? None of other icon is gray.

File: src/styles/theme/defaultTheme.ts [hunk 1]
```diff
@@ -787,16 +787,16 @@ export const defaultTheme = createTheme({
         dot: ({ theme }) => ({
           width: '30px',
           height: '4px',
-          background: theme.palette.layout.secondary,
+          // background: theme.palette.layout.secondary,
```
Comment (buberdds) [inline src/styles/theme/defaultTheme.ts:790]: leftover

File: src/app/pages/HomePage/Graph/HelpScreen/index.tsx [hunk 2]
```diff
@@ -51,33 +53,34 @@ const iconSx = (theme: Theme) => ({
...
 }
 
 const localStore = storage()
 
-const HelpScreen: FC<HelpScreenProps> = ({ setParaTimeStep }) => {
+const HelpScreen: FC<HelpScreenProps> = ({ network, setParaTimeStep }) => {
   const { t } = useTranslation()
   const [activeStep, setActiveStep] = useState<AvailableSteps>(0)
-  const allSteps = steps(t)
+  const allSteps = steps(t, network === Network.mainnet ? COLORS.aqua : COLORS.brandExtraDark)
```
Comment (buberdds) [inline src/app/pages/HomePage/Graph/HelpScreen/index.tsx:83]: I would control this via theme layout colors rather than network prop.

**Review by buberdds — APPROVED** [review]

**Comment by lubej** [comment]
> Does tap/finger icon have a correct color? None of other icon is gray.

Double checked. Should be good.

**Pull request merged**